gdx/src/com/badlogic/gdx/scenes/scene2d/ui/SelectBox.java

```diff
@@ -240,11 +241,27 @@ else if (style.background != null)
 				y += (int)(height / 2 + font.getData().capHeight / 2);
 			}
 			font.setColor(fontColor.r, fontColor.g, fontColor.b, fontColor.a * parentAlpha);
-			layout.setText(font, string, 0, string.length(), font.getColor(), width, Align.left, false, "...");
+			layout.setText(font, string, 0, string.length(), font.getColor(), width, selectedAlignment, false, "...");
 			font.draw(batch, layout, x, y);
 		}
 	}
 
+	/**
+	 * Sets the alignment of the items in the list.
+	 * @param listAlignment The alignment. Use Align constants.
+	 */
+	public void setListAlignment(int listAlignment){
+		this.getList().setListAlignment(listAlignment);
+	}
+
+	/**
+	 * Sets the alignment of the selected element.
+	 * @param alignment The alignment. Use Align constants.
+     */
+	public void setSelectedAlignment(int alignment){
+		this.selectedAlignment = alignment;

```

Comment:
Do we really need 2 alignments? I'd expect the selected item to use the same alignment as the list.